### DISCUSSION TITLE
sync error in lineage 18.1

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#9033] [Q&A]
getting sync error in lineage 18.1 tried several times still getting the same error 

https://cirrus-ci.com/task/5558807974641664
https://cirrus-ci.com/task/5223786701127680

my local manifest looks fine = https://github.com/SwapnilVicky/local_manifest/blob/LOS/local_manifest.xml

**Comment by Apon77** [comment]
> getting sync error in lineage 18.1 tried several times still getting the same error
> 
> https://cirrus-ci.com/task/5558807974641664 https://cirrus-ci.com/task/5223786701127680
> 
> my local manifest looks fine = https://github.com/SwapnilVicky/local_manifest/blob/LOS/local_manifest.xml

Please check your sync script locally first.
